crypto/fipsmodule/rand/rand.c

```diff
@@ -65,26 +65,28 @@
 
 #if defined(BORINGSSL_FIPS)
 
-// In the future this will be demoted to a build-time option.
-// Another entropy source will be promoted to the default source.
-#define FIPS_ENTROPY_SOURCE_JITTER_CPU 1
-
-#if !defined(FIPS_ENTROPY_SOURCE_JITTER_CPU)
-#error "Currently, Jitter CPU must be configured as the entropy source in FIPS mode"
+#if defined(FIPS_ENTROPY_SOURCE_JITTER_CPU) && defined(FIPS_ENTROPY_SOURCE_PASSIVE)
+#error "Only one FIPS entropy source must not be defined"
```

Comment:
```suggestion
#error "Only one FIPS entropy source can be enabled at a time"
```

Reply:
not sure what my original sentence is actually trying to say. Will corrrect.